cmd/sovereignnode/systemTestDemo/mockNotifier/notifier.go

```diff
@@ -289,43 +298,89 @@ func createOutGoingBridgeOpsConfirmationLogs(confirmedBridgeOps []*ConfirmedBrid
+	esdtTokenData = append(esdtTokenData, numberToBytes(uint64(len(amount)), lenSize)...)     // length of amount
+	esdtTokenData = append(esdtTokenData, amount...)                                          // amount
+	esdtTokenData = append(esdtTokenData, []byte{0x00}...)                                    // not frozen
+	esdtTokenData = append(esdtTokenData, numberToBytes(uint64(len(hash)), lenSize)...)       // length of hash
+	esdtTokenData = append(esdtTokenData, hash...)                                            // hash
+	esdtTokenData = append(esdtTokenData, numberToBytes(uint64(len(name)), lenSize)...)       // length of name
+	esdtTokenData = append(esdtTokenData, name...)                                            // name
+	esdtTokenData = append(esdtTokenData, numberToBytes(uint64(len(attributes)), lenSize)...) // length of attributes
+	esdtTokenData = append(esdtTokenData, attributes...)                                      // attributes
+	esdtTokenData = append(esdtTokenData, creator...)                                         // creator
+	esdtTokenData = append(esdtTokenData, numberToBytes(uint64(len(royalties)), lenSize)...)  // length of royalties
+	esdtTokenData = append(esdtTokenData, royalties...)                                       // royalties
+	esdtTokenData = append(esdtTokenData, numberToBytes(uint64(len(uris)), lenSize)...)       // number of uris
+	for _, uri := range uris {
+		esdtTokenData = append(esdtTokenData, numberToBytes(uint64(len(uri)), lenSize)...) // length of uri
+		esdtTokenData = append(esdtTokenData, []byte(uri)...)                              // uri
 	}
 
-	return marshaller.Marshal(esdtData)
+	return esdtTokenData
+}
+
+func createEventData(nonce uint64, addr []byte) []byte {
+	gasLimit := uint64(10000000)
+	function := []byte("func")
+	args := [][]byte{[]byte("arg1")}
+
+	eventData := make([]byte, 0)
+	eventData = append(eventData, numberToBytes(nonce, u64Size)...)                 // event nonce
+	eventData = append(eventData, addr...)                                          // original sender
+	eventData = append(eventData, []byte{0x01}...)                                  // has transfer data
+	eventData = append(eventData, numberToBytes(gasLimit, u64Size)...)              // gas limit bytes
+	eventData = append(eventData, numberToBytes(uint64(len(function)), lenSize)...) // length of function
+	eventData = append(eventData, function...)                                      // function
+	eventData = append(eventData, numberToBytes(uint64(len(args)), lenSize)...)     // number of arguments
+	for _, arg := range args {
+		eventData = append(eventData, numberToBytes(uint64(len(arg)), lenSize)...) // length of current argument
+		eventData = append(eventData, arg...)                                      // current argument
+	}
+
+	return eventData
+}
+
+func numberToBytes(number uint64, size int) []byte {
```

Comment:
I think we could've used bigEndianConverter here if that would work...
no strong opinion, can stay as it is